docs/en/docs/lakehouse/multi-catalog/hive.md

```diff
@@ -56,10 +56,13 @@ CREATE CATALOG hive PROPERTIES (
 
  In addition to `type` and  `hive.metastore.uris` , which are required, you can specify other parameters regarding the connection.
 
-> `specified_database_list`:
-> 
-> only synchronize the specified databases, split with ','. Default values is '' will synchronize all databases. db name is case sensitive.
-> 
+> `include_database_list`:
```

Comment:
This is a common property, you can put it in this doc: https://doris.apache.org/zh-CN/docs/dev/lakehouse/multi-catalog/
So that you don't need to write it for every catalog type

Reply:
OK, I'll move docs in hive.md and iceberg.md to multi-catalog.md.
And I think we should keep jdbc.md since config like `include_database_list` should work with `only_specified_database`